Source/Particles/Deposition/CurrentDeposition.H

```diff
@@ -367,9 +327,7 @@ void doDepositionShapeN (const GetParticlePosition<PIdx>& GetPosition,
 
             doDepositionShapeNKernel<depos_order>(xp, yp, zp, wq, vx, vy, vz, jx_arr, jy_arr, jz_arr,
                                                   jx_type, jy_type, jz_type,
-                                                  relative_time,
-                                                  AMREX_D_DECL(dzi, dxi, dyi),
```

Comment:
The switch to `amrex::XDim3` is particularly nice here to avoid this confusing order of `z, x, y`.